### PR TITLE
[server] Update more GitLab integration settings URLs

### PR DESCRIPTION
**Pull request by gtsiolis**
This will update some more GitLab integretion settings URLs following the updates in https://github.com/gitpod-io/gitpod/pull/4462. 🦊

### CODE REVIEW

File: components/server/src/gitlab/gitlab-urls.ts [hunk 1]
```diff
@@ -9,6 +9,6 @@ export function oauthUrls(host: string) {
     return {
         authorizationUrl: `https://${host}/oauth/authorize`,
         tokenUrl: `https://${host}/oauth/token`,
-        settingsUrl: `https://${host}/profile/applications`,
+        settingsUrl: `https://${host}/-/profile/applications`,
```
Comment (gtsiolis) [inline components/server/src/gitlab/gitlab-urls.ts:12]: This is probably the single most important change here. 😇

**Comment by corneliusludmann** [comment]
FYI: There are also 2 occurrences in our [gitpod-com repo](https://github.com/gitpod-com/gitpod/search?q=profile%2Fapplications). However, I don't think they are crucial.

**Review by corneliusludmann — APPROVED** [review]
Looks good.

**Comment by gtsiolis** [comment]
Thanks for taking a look @corneliusludmann! Merging this and adding a PR for updating those other 2 occurrences.

**Pull request merged**